ajax/search.php

```diff
@@ -63,7 +63,7 @@
         }
 
         /** @var CommonDBTM $itemtype */
-        $itemtype = $_REQUEST['itemtype'];
+        $itemtype = $_UREQUEST['itemtype'];
```

Comment:
This should not be required.

This piece of code should handle this. Maybe a fix is required, or maybe your class is not loaded correctly at this point.
https://github.com/glpi-project/glpi/blob/f8dd0f13d15b45dd1428e9dcd1ec4b7e6e3f3e44/src/Toolbox/Sanitizer.php#L74-L77

Reply:
In `inc/includes.php` file, sanitization process should be moved after plugins loading.